site/menuTest.html

```diff
@@ -0,0 +1,24 @@
+--- 
+--- 
+<!DOCTYPE html>
+<!--
+    Test page for working on menus.
+-->
+<html lang=>
+    {% include head.html %}
+<body class="layout--{{ page.layout | default: layout.layout }}{% if page.classes or layout.classes %}{{ page.classes | default: layout.classes | join: ' ' | prepend: ' '}}{% endif %} {{ page.title | slugify }}">
```

Comment:
The rest of the code is fairly easy to follow, but this line is too complex.  This is a test file, which won't be included anyways.